vm/src/bench.rs

```diff
@@ -151,3 +170,402 @@ fn trace_as_trace(fvm_trace: fvm::trace::ExecutionTrace) -> ExecutionTrace {
     }
     ExecutionTrace::new(events)
 }
+#[derive(Debug, Clone)]
+pub struct TestData {
+    circ_supply: TokenAmount,
+    price_list: PriceList,
+}
```

Comment:
Name this something better, like TestConfiguration, or Fakes or similar. I imagine it will expand with configuration for every method that is not forwarded. In fact, maybe it should be a trait instead (with default impls), so test code can intercept every time this is consulted rather than provide only static values.